athena/evaluation/service/json_service.py

```
@@ -601,3 +604,250 @@ def extract_metrics_and_mappings(data):
+                        record = {
+                            "expert_id": expert_id,
+                            "exercise_id": exercise_id,
+                            "submission_id": submission_id,
+                            "feedback_type": feedback_type,
+                            "metric": metric,
+                            "score": score,
+                        }
+                        records.append(record)
+    
+    if not records:
+        raise ValueError("No valid evaluation progress records found.")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)